reddit_about/templates/values.html

```diff
@@ -0,0 +1,73 @@
+<div class="container">
+  <div class="image-container">
+    <div class="angel-snoo-graphic"></div>
+  </div>
+  <div class="core-values-container">
+    <ul class="core-values">
```

Comment:
This would make it more painful to deal with, but I wonder if we should consider pulling this from a wiki page? That would make it so that we can easily make changes over time and also maintain a public diff of the changes (which may be interesting) on the site.

Probably fairly optional, especially considering this repo itself is public and we have a history of this file.